docs/src/main/sphinx/connector/hive-security.rst

```diff
@@ -519,6 +519,31 @@ These rules govern the privileges granted on specific tables.
 
 * ``filter_environment`` (optional): environment use during filter evaluation.
 
+.. note::
+
+    ``OWNERSHIP`` privilege is required for performing the following operations::
```

Comment:
Maybe have a title above this like "Ownership requirement"

Reply:
Might actually want to make this a section on `Privileges` with subheadings `SELECT`, `INSERT`, `DELETE`, `GRANT_SELECT` and `OWNSERSHIP`.
the new content can go under `OWNERSHIP`.

Reply:
> Might actually want to make this a section on `Privileges` with subheadings `SELECT`, `INSERT`, `DELETE`, `GRANT_SELECT` and `OWNSERSHIP`.
> the new content can go under `OWNERSHIP`.

I think it is a good suggestion, @mosabua how about you? 
The question is where and how we make this section, do we have some good or similar demo in our docs, i think i need learn first then start to work.

Reply:
Sounds good